src/components/MarkdownMusicSourceFetcher.js

```diff
@@ -30,7 +32,7 @@ class MarkdownMusicSourceFetcher extends React.Component {
     const repo = this.props.match.params.repo;
     const path = this.props.match.params.path;
 
-    getRepoDirectoryList(owner, repo, path)
```

Comment:
This is still necessary for later when we pull contents from a repo, keep in for now.